pkg/cmd/roachtest/roachtestutil/mixedversion/runner.go

```diff
@@ -518,7 +518,7 @@ func (tr *testRunner) refreshBinaryVersions(ctx context.Context, service *servic
 	defer cancel()
 
 	group := ctxgroup.WithContext(connectionCtx)
-	for j, node := range service.descriptor.Nodes {
+	for j, node := range tr.monitor.AvailableNodes(install.SystemInterfaceName) {
```

Comment:
Shouldn't we use `service.AvailableNodes`? That way you don't need to specify the virtual cluster name.